.rubocop.yml

```diff
@@ -776,12 +777,21 @@ Rails/HttpPositionalArguments:
     - spec/**/*
     - test/**/*
 
+Rails/I18nLocaleAssignment:
+  Enabled: false
```

Comment:
this wanted us to use the block form `I18n.with_locale` everywhere, which just isn't realistic for us to use